platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/LibraryLoader.java

```diff
@@ -1,15 +1,38 @@
 package com.mapbox.mapboxsdk;
 
+import android.content.Context;
+
+import java.io.File;
+
 /**
  * Centralises the knowledge about "mapbox-gl" library loading.
  */
 public class LibraryLoader {
 
+  private static final String LIBRARY_NAME = "libmapbox-gl.so";
+
   /**
    * Loads "libmapbox-gl.so" native shared library.
+   * @param context The application context
    */
-  public static void load() {
-    System.loadLibrary("mapbox-gl");
+  public static void load(Context context) {
```

Comment:
we could keep the signature of this the same if we replace the context below with `Mapbox.getApplicationContext()`?

Reply:
Good idea. I did that with https://github.com/mapbox/mapbox-gl-native/pull/10154/commits/1d4c6752225f9fe91c24ceded08ed1d586444bd0, I'll merge it once CI gives the blessing.